algo/src/main/java/org/neo4j/graphalgo/SCCTarjanProc.java

```diff
@@ -0,0 +1,73 @@
+package org.neo4j.graphalgo;
+
+import algo.Pools;
+import org.neo4j.graphalgo.api.Graph;
+import org.neo4j.graphalgo.core.GraphLoader;
+import org.neo4j.graphalgo.core.heavyweight.HeavyGraphFactory;
+import org.neo4j.graphalgo.core.utils.ProgressTimer;
+import org.neo4j.graphalgo.impl.SCCTarjan;
+import org.neo4j.graphalgo.impl.SCCTarjanExporter;
+import org.neo4j.graphalgo.results.SCCResult;
+import org.neo4j.kernel.internal.GraphDatabaseAPI;
+import org.neo4j.logging.Log;
+import org.neo4j.procedure.*;
+
+import java.util.Map;
+import java.util.stream.Stream;
+
+/**
+ * @author mknblch
+ */
+public class SCCTarjanProc {
+
+    public static final String CONFIG_WRITE = "write";
+    public static final String CONFIG_PROPERTY = "property";
+    public static final String CONFIG_DEFAULT_VALUE = "defaultValue";
+    public static final String CONFIG_WRITE_PROPERTY = "clusterProperty";
+    public static final String CONFIG_CLUSTER = "cluster";
+
+    @Context
+    public GraphDatabaseAPI api;
+
+    @Context
+    public Log log;
+
+    @Procedure(value = "algo.scctarjan", mode = Mode.WRITE)
```

Comment:
the algorithm procedure should just be called algo.scc or algo.stronglyConnectedComponents

Tarjan is an implementation detail.